src/CommunityToolkit/Xamarin.CommunityToolkit/Views/RangeSlider/SegmentedView/SegmentMode.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
```

Comment:
Why this file is inside RangeSlider folder ?

Reply:
This is dumb. I saw it was moved by fat fingering the dragging. I will fix it, again :)